devserver.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Same as travis deploy script
+
+# npm install
+npm run build
+rm -r static/
+mkdir static/
+cp -r ./dist/dev/* static/
+virtualenv env
+source env/bin/activate
+pip install -t lib -r requirements.txt
+dev_appserver.py app.yaml
```

Comment:
I was having trouble getting build-min to work, something about EPIPE. I also omitted several travis related items here and added the dev_appserver.py line. There's a core here that's common, but I don't think I want to integrate them just yet.